src/test/home.test.js

```diff
@@ -0,0 +1,22 @@
+// import covidCasesReducer from '../redux/covid/covid-cases-reducer';
+// import setCovidCases from '../redux/covid/covid-cases-actions';
```

Comment:
- [ ] Please kindly remove commented code from your code base as it makes your code dirty and less readable.

Reply:
Okay. Done